test_script.py

```diff
@@ -0,0 +1,134 @@
+import os
```

Comment:
@manangoel99 Will you remove this test script or do you still have things to test or have open questions?

Reply:
Otherwise the PR looks good to me